INSTALL_FROM_SOURCE.md

```diff
@@ -81,7 +81,7 @@ brew install libsndfile
 
 ### 3.2 PyTorch
 Follow the instructions on [pytorch.org](https://pytorch.org) to install the
-desired PyTorch version. Note that fairseq2 supports only PyTorch 1.12 or
+desired PyTorch version.
```

Comment:
Shouldnt we specify the min version it supports?

Reply:
I am working on PyTorch 2.1 builds right now, and plan to consolidate all version information in README to reduce the maintenance overhead.

Reply:
Ok, put a link here to the supported version list in README (see latest version). Thanks again for the feedback!

Reply:
Thank you!